e2e/pipelines/test_generative_qa.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pytest
+
+from haystack import Document
+from haystack.pipelines import TranslationWrapperPipeline, GenerativeQAPipeline
+from haystack.document_stores import InMemoryDocumentStore
+from haystack.nodes import BM25Retriever, RAGenerator, TransformersTranslator
+
+
+from ..conftest import SAMPLES_PATH
+
+
+@pytest.fixture
+def docs_with_true_emb():
```

Comment:
Should this fixture be defined here? I see it's still used in `test/nodes/test_generator.py`, probably it should be in a `conftest.py`?

Reply:
There's no shared `conftest.py` across `e2e` and `tests` (and to be really honest I don't want one to exist...)

However, you make me notice that this fixture seems unused here. Must have been a leftover... I'm removing it.